#!/usr/bin/env python
PACKAGE = "uav_ros_lib"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("noise_pos_mv",     double_t,    0,     "Kalman Filter measured position value stddev",          1,    0,  500)
gen.add("noise_acc_mv",     double_t,    0,     "Kalman Filter measured acceleration value stddev",          1,    0,  500)
gen.add("noise_pos_proc",    double_t,    0,     "Kalman Filter position process noise",                1,    0,   500)
gen.add("noise_vel_proc",    double_t,    0,     "Kalman Filter velocity process noise",           1,    0,  500)
gen.add("noise_acc_proc",    double_t,    0,     "Kalman Filter velocity process noise",           1,    0,  500)

exit(gen.generate(PACKAGE, "uav_ros_lib", "ConstantAccelerationLKFParameters"))